### ISSUE TITLE
web-wallet: no matching manifest for linux/arm64/v8 in the manifest list entries

### ISSUE THREAD

**Issue by aridder** [walt-id/waltid-identity#54]
System:

Macbook pro M2

Trying to run the web-wallet and following [this](https://github.com/walt-id/waltid-identity/tree/main/waltid-web-wallet) guide.

```bash
➜  docker-compose git:(main) ✗ docker compose up
[+] Running 0/21
 ⠧ vc-repo Pulling                                                                                                           1.7s
 ⠧ ingress Pulling                                                                                                           1.7s
 ⠧ web-portal 8 layers [⠀⠀⠀⠀⠀⠀⠀⠀]      0B/0B      Pulling                                                                    1.7s
   ⠼ 661ff4d9561e Waiting                                                                                                    0.4s
   ⠼ 89059ca18a98 Waiting                                                                                                    0.4s
   ⠼ 9bb776f8ddb0 Waiting                                                                                                    0.4s
   ⠼ ef0b6390b7f1 Waiting                                                                                                    0.4s
   ⠼ a15a9d425880 Waiting                                                                                                    0.4s
   ⠼ 1e712b2c24dd Waiting                                                                                                    0.4s
   ⠼ 8c46756e133c Waiting                                                                                                    0.4s
   ⠼ 9a8e93e2029b Waiting                                                                                                    0.4s
 ⠧ verifier-api 7 layers [⠀⠀⠀⠀⠀⠀⠀]      0B/0B      Pulling                                                                   1.7s
   ⠼ 3dd181f9be59 Pulling fs layer                                                                                           0.4s
   ⠼ 0f838805bddf Pulling fs layer                                                                                           0.4s
   ⠼ dfab58ae8b86 Pulling fs layer                                                                                           0.4s
   ⠼ c6cc1bdab538 Waiting                                                                                                    0.4s
   ⠼ cc590439cbf2 Waiting                                                                                                    0.4s
   ⠼ fca4b39976e3 Waiting                                                                                                    0.4s
   ⠼ 4f4fb700ef54 Waiting                                                                                                    0.4s
 ⠦ issuer-api 1 layers [⠀]      0B/0B      Pulling                                                                           1.7s
   ⠼ b6a121b98e02 Waiting                                                                                                    0.4s
no matching manifest for linux/arm64/v8 in the manifest list entries
```

**Comment by dinkar-jain** [comment]
Should work now with the latest changes. Please do a `git pull` before retrying.

**Comment by aridder** [comment]
Can confirm this fixed my issue. But a new rised. Fixed in #55

**Comment by aridder** [comment]
Yes thank you.
